src/common/transformations/include/transformations/op_conversions/gathernd_decomposition.hpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
```

Comment:
Please take performance measurements to understand how much gain this transformation delivers.
Also, since you move this transformation to offline transformations, will you remove it from the Model Optimizer? I think it makes sense to do in the scope of this PR as well.

Reply:
I am in progress of getting performance data. Regarding removal of `GatherNDDecomposition` from MO, I believe not all frontends use the new API - for example TensorFlow is still using only MO transformations. Please correct me if I'm wrong.

Reply:
No, models converted from legacy TF are also going through common transformations. Are you getting performance data on real model or synthetic one?

Reply:
Now we have several questions about this transformation:
1. What real model (from our scope) does need this transformation? 
2. If there exists the model, which plugin does need this transformation due to no support of GatherND there? If yes, then we need this transformation offline
3. Does this transformation deliver any performance gain? If yes, we need this transformation offline

Reply:
I removed this transformation from Model Optimizer. Here is an example model which has GatherND nodes and could be transformed, I will run benchmark app on it to see if there is performance gain, as well as dlbenchmark on OMZ models. I do not yet have an answer for the question about plugins.